tests/backend/base/FunctionalTestCase.php

```diff
@@ -441,9 +441,7 @@ private function hasValidDateFormat($dateString): bool
      */
     protected function getProcedurePhases()
     {
-        $path = DemosPlanPath::getRootPath('demosplan/DemosPlanProcedureBundle/Resources/config');
-
-        return Yaml::parseFile($path.'/procedurephases.yml');
+        return Yaml::parseFile(DemosPlanPath::getProjectPath('app/Resources/DemosPlanCoreBundle/config/procedurephases.yml'));
```

Comment:
Why did you move this to the project? Shouldn't this be `DemosPlanPath::getConfigPath('procedure/procedurephases.yml')`?

Reply:
you are right, this was a mistake, is fixed in https://github.com/demos-europe/demosplan-core/pull/1408/commits/18727d3cd76cf634764de38ebe685828eab2630d